internals/generators/container/index.js

```diff
@@ -20,10 +20,10 @@ module.exports = {
     },
   }, {
     type: 'list',
-    name: 'component',
-    message: 'Select a base component:',
-    default: 'PureComponent',
-    choices: () => ['PureComponent', 'Component'],
+    name: 'type',
+    message: 'Select the base component type:',
+    default: 'Stateless Function',
+    choices: () => ['Stateless Function', 'PureComponent', 'Component'],
```

Comment:
👍  Agree on 'React.Component', 'React.PureComponent' and 'Stateless Function'.

Reactland has a naming problem with 'Component', JS vs CSS has a problem with 'Class', plain JS functions as 'Functional Components' has a problem with 'Functional Programming', etc. 

It's important (and often hard) to name things correctly 😄